ledger/accountdb.go

```diff
@@ -193,7 +193,8 @@ type persistedOnlineAccountData struct {
 	rowid       int64
 	// the round number that is associated with the baseOnlineAccountData. This field is the corresponding one to the round field
 	// in persistedAccountData, and serves the same purpose.
-	round basics.Round
+	round    basics.Round
+	updRound basics.Round
```

Comment:
Agreed, it would be nice to make clear that the `round` column always comes from acctrounds (indicating what DB round this data was written) versus the `updRound` which is the actual round that the online account data is relevant from ... the other persistedXData types don't have two round numbers like this.